app/javascript/components/components/Header.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react'
+import {
+  Navbar,
+  NavItem,
+  NavLink,
+  NavbarToggler,
+  NavbarBrand,
+  Collapse,
+  Nav,
+  NavbarText} from 'reactstrap';
+
+class Home extends Component {
+  render() {
+    return(
+    <>
+    <div>
+  <Navbar
+    color="light"
+    expand="md"
+    light
+  >
+    <NavbarBrand href="http://localhost:3000">
+      Apartment App
+    </NavbarBrand>
+    <NavbarToggler onClick={function noRefCheck(){}} />
+    <Collapse navbar>
+      <Nav
+        className="me-auto"
+        navbar
+      >
+        <NavItem>
+          <NavLink href="http://localhost:3000/apartmentindex">
+            Listings
```

Comment:
NavLInks to internal React pages should not have an href. 
You defined a route called `<Route path="/apartmentindex" component={ApartmentIndex} />` You can just call the route you defined.